setup.py

```diff
@@ -20,6 +20,7 @@
         path = line.split()[-1]
         break
 
+path=os.environ['CONDA_PREFIX']
```

Comment:
This isn't going to work generally as it's specific to conda environments and even in conda-forge builds this will be `PREFIX` not `CONDA_PREFIX`. 

I'll open a PR to your fork with a more generic way to grab the include and library paths.